.github/workflows/ci-tests/Examples_tests/test_launcher.sh

```diff
@@ -106,6 +147,27 @@ function flash_with_openocd() {
         openocd_dapLink_pid=$!
     fi
 }
+
+#****************************************************************************************************
+# params: target in lower case, DAP sn
+function reset_board_by_openocd() {
+    echo "function: ${FUNCNAME[0]} $@"
+
+    set +e
+    $OPENOCD -f $OPENOCD_TCL_PATH/interface/cmsis-dap.cfg \
+             -f $OPENOCD_TCL_PATH/target/$1.cfg -s $OPENOCD_TCL_PATH \
+             -c "adapter serial $2" \
+             -c "gdb_port 3333" -c "telnet_port 4444" -c "tcl_port 6666" \
+             -c "init; reset run" >/dev/null &
```

Comment:
should this maybe have the "exit" command at the end ? I thought openocd would not exit without it

Reply:
"exit" works with "program" cmd.